docs/source/guide/zne-5-theory.md

```diff
@@ -50,11 +52,26 @@ name: figzne_local
 ---
 The diagram demonstrates how gates are inserted in a circuit when *local folding* is applied.
 ```
-
 More details on the theory of unitary folding can be found in {cite}`Giurgica_Tiron_2020_arXiv`.
-More details on its practical implementation in Mitiq can be found in 
+
+In *identity insertion scaling*, we perform a mapping $G \mapsto I G$. 
+This mapping can be applied as shown in the diagram below.
+
+```{figure} ../img/zne_id_scaling_layers.png
+---
+width: 500
+name: figid
+---
+The diagram demonstrates how identity gates are inserted after a circuit layer when identity insertion scaling function is applied.
+```
+
+Additional details on the theory of identity insertion scaling are similar to those in unitary folding. The only difference 
+is that instead of scaling gate noise, the insertion of an identity gate increases the wait time after each circuit layer is executed. This allows the qubits to interact with the environment through some noisy process. The decoherence time for the qubits in a quantum system is determined by the amount of time our system of interest remains coherent and uncouples from the external environment. More details on decoherence can be found in {cite}`Schlosshauer_2019, Zurek_2003_arxiv`.
```

Comment:
```suggestion
is that instead of scaling gate noise, the insertion of an identity gate increases the wait time after each circuit layer is executed. This allows the qubits to interact with the environment through some noisy process and decohere if the system-environment interaction is strong. The decoherence time for the qubits in a quantum system is determined by the amount of time our system of interest remains coherent and uncouples from the external environment. More details on decoherence can be found in {cite}`Schlosshauer_2019, Zurek_2003_arxiv`.
```